arch/arm/boot/dts/nxp/imx/imx6ul-ts7250v3.dtso

```diff
@@ -0,0 +1,36 @@
+/dts-v1/;
+/plugin/;
+
+/ {
+    compatible = "embeddedts,ts-7250-v3";
+
+    fragment@0 {
+        target-path = "/";
+        __overlay__ {
+            i2c5_gpio: i2c-104 {
+                compatible = "i2c-gpio";
+
+                /* FPGA bank1 pins: SCL=13, SDA=8 */
+                scl-gpios = <&fpga_bank1 13 0>;
+                sda-gpios = <&fpga_bank1  8 0>;
+
+                /* Open-drain for both lines (legacy property name for broad kernel support) */
+                i2c-gpio,open-drain;
+
+                /* Optional timing tweak: */
+                // i2c-gpio,delay-us = <5>;
+
+                #address-cells = <1>;
+                #size-cells = <0>;
+                status = "okay";
```

Comment:
`okay` is the default status. While I'm not familiar enough with devicetree overlays, that should still apply here and the status line should not be needed.